src/libraries/System.Linq/tests/SingleOrDefaultTests.cs

```diff
@@ -45,6 +54,15 @@ public void SingleElementIList()
             Assert.Equal(expected, source.SingleOrDefault());
         }
 
+        [Fact]
+        public void SingleElementIListDefault()
```

Comment:
`SingleOrDefault` containing more than one elements should throw. Please include a test that verifies that behaviour.